src/adv.py

```diff
@@ -49,3 +61,33 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+
+
+
+def get_items_from_room(player_selected_itemnames, room, player):
+    for player_selected_itemname in player_selected_itemnames:
+        if not room.is_item_in_room(player_selected_itemname):
+            print(f"Item not found: {player_selected_itemname}")
+            return input(f"(Press enter to continue...)")
+
+    player.add_items(room.get_items(player_selected_itemnames))
+
+    print(f"{player} stores items: {player_selected_itemnames}")
+    return input(f"(Press enter to continue...)")
+
+
+while True:
+    print(player.room.name)
+    print(player.room.description)
+    print(player.room.items)
+    # cmd = input("What do you want to do? ")
+    cmd = ' '.join(input("Enter command: ").lower().split()).split(" ")
+
+    if cmd[0] in ['n', 'e', 's', 'w']:
+        player.travel(cmd[0])
+    elif "get" in cmd[0] or "take" in cmd[0]:
+        get_items_from_room(cmd[1:], player.room, player)
+    elif cmd[0] == 'q':
+        break
+    else:
+        print('invalid command')
```

Comment:
Nice job on keeping this while loop short and clean. It's good to separate the code blocks into functions. Makes the code much more readable.